roles/django/templates/deployed.j2

```diff
@@ -6,6 +6,11 @@ from pathlib import Path
 
 from .base import *  # noqa
 
+FORCE_SCRIPT_NAME = '/'
```

Comment:
Given that this worked before without these overrides, I'd be tempted instead to revert these changes, and just say in the readme that suitable lines need to be *added*.

Reply:
yeah ok